### ISSUE TITLE
Github-Cli auth not working

### ISSUE THREAD

**Issue by heathercat123** [vxiiduu/VxKex#276]
Alright so I wanted to login to github using git + github-cli on windows 7 but instead, i got this: 
```
√ Authentication complete.
- gh config set -h github.com git_protocol https
√ Configured git protocol
failed to run git: fork/exec C:\Program Files\Git\cmd\git.exe: Insufficient system resources exist to complete the requested service.
```

The command i executed was ```gh auth login```.

note: i'm pretty new to github pls don't scream at me if i did smth wrong-

**Comment by CopperFr** [comment]
You didn't do anything wrong as it needed some changes in VxKex to make it works...

**Comment by heathercat123** [comment]
> You didn't do anything wrong as VxKex needed some changes to make it works...

Oh well. I hope it'll work someday!

**Comment by CopperFr** [comment]
You can try with my new version...

[VxKex.zip](https://github.com/user-attachments/files/16048677/VxKex.zip)

In your case I think you only need to replace KxBase.dll...

**Comment by i486** [comment]
@CopperFr Was this issue related to console handle?

**Comment by CopperFr** [comment]
Yes I added the filter to remove them which was removed from Go Lang

**Comment by i486** [comment]
> Yes I added the filter to remove them which was removed from Go Lang

It took me days to figure out that the `Insufficient system resources exist to complete the requested service.` error was caused by the removal of a console handle workaround from a Golang PR. I had to recompile it with the patch reverted to fix it. Now, thanks to your work, no one has to do that anymore. Keep up the fantastic work!
